scene/gui/popup_menu.cpp

```diff
@@ -313,17 +316,31 @@ int PopupMenu::_get_items_total_height() const {
 }
 
 int PopupMenu::_get_mouse_over(const Point2 &p_over) const {
+	// Make the item area exclude shadows and the vertical margins and scrollbar.
+	Rect2 item_clickable_area = panel->get_global_rect();
+	if (scroll_container->get_v_scroll_bar()->is_visible_in_tree()) {
+		const int scroll_width = scroll_container->get_v_scroll_bar()->get_size().width;
+		if (is_layout_rtl()) {
+			item_clickable_area.position.x += scroll_width;
+			item_clickable_area.size.width -= scroll_width;
+		}
+		item_clickable_area.size.width -= scroll_width;
+	}
 	float win_scale = get_content_scale_factor();
-	if (p_over.x < 0 || p_over.x >= get_size().width * win_scale || p_over.y < theme_cache.panel_style->get_margin(Side::SIDE_TOP) * win_scale) {
+	item_clickable_area.position.y = (item_clickable_area.position.y + theme_cache.panel_style->get_margin(SIDE_TOP)) * win_scale;
+	item_clickable_area.size.y -= theme_cache.panel_style->get_margin(SIDE_TOP) + theme_cache.panel_style->get_margin(SIDE_BOTTOM);
+	item_clickable_area.size *= win_scale;
+
+	if (p_over.x < item_clickable_area.position.x || p_over.x >= item_clickable_area.position.x + item_clickable_area.size.width ||
+			p_over.y < item_clickable_area.position.y || p_over.y >= item_clickable_area.position.y + item_clickable_area.size.height) {
```

Comment:
Would this be equivalent to:
```suggestion
	if (!item_clickable_area.has_point(p_over)) {
```

Reply:
Integrated into #102625.